tools:
  capture_and_score: 1
  mast_rotate: 2
  mast_open: 2
  mast_close: 2
  move_forward: 5
  turn_left: 2
  turn_right: 2
  move_stop: 1
  get_status: 1
